src/kOS/Suffixed/WaypointValue.cs

```diff
@@ -8,47 +9,80 @@ namespace kOS.Suffixed
 {
     public class WaypointValue : Structure
     {
-        protected Waypoint WayPoint { get; set; }
+        protected Waypoint WrappedWaypoint { get; set; }
```

Comment:
Renamed purely for style reasons: If the type name "Waypoint" establishes the standard that the "P" in "Point" doesn't count as a second word (not capitalized), then that should be adhered to universally, and that would make the property ```WayPoint``` into just ```Waypoint```.  I then renamed ```Waypoint``` to ```WrappedWaypoint``` because although C# allows you to use the exact same identifier (including capitalization) as both a type name and an instance name, I consider that bad practice and try to avoid it, but the capitalization rules the project is operating under don't allow this property to be lowercase named (because it uses capitalization to differentiate fields from methods rather than the standard I'm used to which uses it to differentiate type names from instance names), so I was left with just giving it a totally different name.  I chose ```WrappedWaypoint``` as in "the waypoint being wrapped by this WaypointValue class"